internal/pkg/aws/cloudformation/stackset/status.go

```diff
@@ -0,0 +1,72 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package stackset
+
+import "github.com/aws/aws-sdk-go/service/cloudformation"
+
+const (
+	opStatusSucceeded = cloudformation.StackSetOperationStatusSucceeded
+	opStatusStopped   = cloudformation.StackSetOperationStatusStopped
+	opStatusFailed    = cloudformation.StackSetOperationStatusFailed
+)
+
+const (
+	instanceStatusPending    = cloudformation.StackInstanceDetailedStatusPending
+	instanceStatusRunning    = cloudformation.StackInstanceDetailedStatusRunning
+	instanceStatusSucceeded  = cloudformation.StackInstanceDetailedStatusSucceeded
+	instanceStatusFailed     = cloudformation.StackInstanceDetailedStatusFailed
+	instanceStatusCancelled  = cloudformation.StackInstanceDetailedStatusCancelled
+	instanceStatusInoperable = cloudformation.StackInstanceDetailedStatusInoperable
+)
+
+// OpStatus represents a stack set operation status.
+type OpStatus string
+
+// IsCompleted returns true if the operation is in a final state.
+func (s OpStatus) IsCompleted() bool {
+	return s.IsSuccessful() || s.IsFailure()
+}
+
+// IsSuccessful returns true if the operation is completed successfully.
+func (s OpStatus) IsSuccessful() bool {
+	return s == opStatusSucceeded
+}
+
+// IsFailure returns true if the operation terminated in failure.
+func (s OpStatus) IsFailure() bool {
+	return s == opStatusStopped || s == opStatusFailed
+}
+
+// InstanceStatus represents a stack set's instance detailed status.
```

Comment:
This file is the same as `opstatus.go` that got deleted, I just renamed the file and added the `InstanceStatus` type